sandy/decay.py

```diff
@@ -287,6 +287,41 @@ def get_decay_chains(self, skip_parents=False, **kwargs):
                .reset_index(drop=True)[columns]
         return df
 
+    def get_chain_yields(self, **kwargs):
+        """
+        Extract chain fission yield matrix into a dataframe
+
+        Parameters
+        ----------
+        kwargs : `dict`
+            keyword arguments for method `get_decay_chains`
+
+        Returns
+        -------
+        `pandas.DataFrame`
+             associated to the given decay chains
+
+        Examples
+        --------
+        >>> zam = [10010, 10020, 10030, 10040]
+        >>> tape = sandy.get_endf6_file("jeff_33",'decay', zam)
+        >>> decay_data = DecayData.from_endf6(tape)
+        >>> decay_data.get_chain_yields()
+        ZAP	      10010	      10020	      10030	      10040	      20030
+        A					
+        1	1.00000e+00	0.00000e+00	0.00000e+00	0.00000e+00	0.00000e+00
+        2	0.00000e+00	1.00000e+00	0.00000e+00	0.00000e+00	0.00000e+00
+        3	0.00000e+00	0.00000e+00	1.00000e+00	0.00000e+00	1.00000e+00
+        4	0.00000e+00	0.00000e+00	1.00000e+00	1.00000e+00	0.00000e+00

```

Comment:
Maybe we need a better example with delayed neutrons.
I wouldn't have implemented the method this way (you don't look at delayed neutrons)...is it correct?

Reply:
ok getting the delayed neutrons from `get_decay_chains`. Still, look inside `get_decay_chains`, check how delayed neutrons are found (maybe add documentation if needed) and see whether there is an alternative/simpler way. If not, keep yours, which is also fine.

For the sensitivity the mathematical formulation is for a given fission product:
- add 1 in correspondence of its mass number A, add 0 elsewhere.
- If the fission product emits a delayed neutron, add 1 in correspondence of mass number A-1 (not for A, because the decay won't lead to A), add 0 elsewhere.
- If it emits 2 delayed neutrons (is it possible?), add 1 in correspondence the end-of-chain mass number A-2 (not for A, not for A-1), 0 elsewhere.
- ...